src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+const treeList = list.querySelectorAll('ul');
+
+list.style.cssText = 'position: absolute ; top: 200px; left 300px';
```

Comment:
you forgot semicolon and left prop wasn't added to the final styles ;)

```suggestion
list.style.cssText = 'position: absolute ; top: 200px; left 300px;';
```